pkg/metrics/resource.go

```diff
@@ -130,14 +137,19 @@ func ObserveSyncWorkloadLatency(err error, start time.Time) {
 	syncWorkloadDurationHistogram.WithLabelValues(utilmetrics.GetResultByError(err)).Observe(utilmetrics.DurationInSeconds(start))
 }
 
-// CountRecreateResourceToCluster records the number of recreating operation of the resource to a target member cluster.
-func CountRecreateResourceToCluster(err error, apiVersion, kind, cluster string) {
-	recreateResourceWhenSyncWorkStatus.WithLabelValues(utilmetrics.GetResultByError(err), apiVersion, kind, cluster).Inc()
+// CountCreateResourceToCluster records the number of recreating operation of the resource to a target member cluster.
+func CountCreateResourceToCluster(err error, apiVersion, kind, cluster string, recreate bool) {
+	createResourceWhenSyncWork.WithLabelValues(utilmetrics.GetResultByError(err), apiVersion, kind, cluster, fmt.Sprint(recreate)).Inc()
```

Comment:
```suggestion
	createResourceWhenSyncWork.WithLabelValues(utilmetrics.GetResultByError(err), apiVersion, kind, cluster, strconv.FormatBool(recreate)).Inc()
```
It seems `strconv.FormatBool(recreate)` is clearer.